code/client/src/core/ui/world_debug.cpp

```diff
@@ -244,6 +248,75 @@ namespace MafiaMP::Core::UI {
             }
         }
 
+        if (ImGui::CollapsingHeader("Racing")) {
+            SDK::mafia::ui::C_GameGUI2Module *GameGuiModule = SDK::mafia::ui::GetGameGui2Module();
+            SDK::ue::C_WeakPtr<SDK::ue::sys::sodb::C_DatabaseInterface> result = GameGuiModule->GetDatabase();
+            if (SDK::mafia::database::C_UIDatabase *database = reinterpret_cast<SDK::mafia::database::C_UIDatabase *>(result.Get())) {
+                SDK::mafia::database::C_UIDatabase::C_HUDTable *hudTable = database->GetHUDTable();
+
+                ImGui::PushItemWidth(75.0f);
+
+                // VISIBLE
+                ImGui::Text("Racing HUDElement Visibility");
+                ImGui::PushID("Racing_Visible");
```

Comment:
Why do you need ID here? (and in other places below)

Reply:
May have been for a previous revision. They can likely be removed.